### DISCUSSION TITLE
Dual Dell Monitors - Mac mini - Double Volume Control

### DISCUSSION THREAD

**Discussion by b0nafide** [MonitorControl/MonitorControl#899] [General]
Hey guys - Great app.

I have dual Dell S2722QC monitors. One is connected to my Mac mini through USB-C and the other is connected through HDMI. 

I have a MIDI profile setup for Both monitors:

<img width="640" alt="Screen Shot 2022-01-19 at 13 05 19" src="https://user-images.githubusercontent.com/97490066/150205529-475112a9-ad93-430b-8428-0ed056688a8f.png">

I installed your app and selected dual volume control. Both monitors appear to lower/raise the volume but only the USB-C monitor changes volume. The HDMI monitor does not change at all (yet shows that it's volume bar is low). It appears that the monitor plugged into HDMI does not have the ability to raise/lower volume?

Anyways, perhaps I have something messed up or there is a different setting I'm not thinking of? 

<img width="236" alt="Screen Shot 2022-01-19 at 13 09 44" src="https://user-images.githubusercontent.com/97490066/150206095-42b6ce4c-73fb-4d17-a7f5-689906942001.png">

As you can see from the quick menu bar, the HDMI monitor is S2722QC (2) but no volume control is enabled. If I select only that monitor as a speaker it still does not allow for volume control - yet shows the volume slider going up/down on screen. 

Any suggestions? 

If no suggestions, is there any way for the app to actually control both volumes at once?

**Comment by waydabber** [comment]
Hi @b0nafide - the HDMI port of the 2018 and 2020 mini do not support DDC control. This is a hardware limitation of the device.
